db/migrate/20151015103504_add_locale_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddLocaleToUsers < ActiveRecord::Migration
+  def change
+    add_column :users, :locale, :string, default: 'en'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.